### PR TITLE
Transit : Update to 3.0.2, and replacement of the conversion script by the transit function

### PR DESCRIPTION
**Pull request by Delphine-L**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/transit/macros.xml [hunk 1]
```diff
@@ -9,11 +9,11 @@
 	<xml name="requirements">
 		<requirements>
 			<requirement type="package" version="@VERSION@">transit</requirement>
-            <requirement type="package" version="3.7">python</requirement>
+            <requirement type="package" version="3.6">python</requirement>
```
Comment (mvdbeek) [inline tools/transit/macros.xml]:
```suggestion
```

Transit comes with python, so you don't need to do this. We've fixed the pillow pinning in https://github.com/bioconda/bioconda-recipes/pull/19837

Comment (davebx) [reply tools/transit/macros.xml]: @mvdbeek it looks like travis is still installing build 0 of transit, without the fix you implemented last week

**Comment by mvdbeek** [comment]
`Checking with container resolver [MulledDockerContainerResolver[namespace=biocontainers]] found description [ContainerDescription[identifier=quay.io/biocontainers/transit:3.0.2--py_1,type=docker]]` that looks good, I think it'll pass now.

File: tools/transit/convert_gff.xml [hunk 2]
```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0"?>
-<tool id="gff_to_prot" name="Convert GFF3" version="1.0.0">
+<tool id="gff_to_prot" name="Convert GFF3" version="@VERSION@+galaxy1">
```
Comment (mvdbeek) [inline tools/transit/convert_gff.xml]:
```suggestion
<tool id="gff_to_prot" name="Convert GFF3" version="@VERSION@+galaxy0">
```

File: tools/transit/transit_gumbel.xml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<tool id="transit_gumbel" name="Gumbel" version="@VERSION@+galaxy2">
+<tool id="transit_gumbel" name="Gumbel" version="@VERSION@+galaxy1">
```
Comment (mvdbeek) [inline tools/transit/transit_gumbel.xml]:
```suggestion
<tool id="transit_gumbel" name="Gumbel" version="@VERSION@+galaxy0">
```

File: tools/transit/transit_hmm.xml [hunk 4]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<tool id="transit_hmm" name="HMM" version="@VERSION@+galaxy2">
+<tool id="transit_hmm" name="HMM" version="@VERSION@+galaxy1">
```
Comment (mvdbeek) [inline tools/transit/transit_hmm.xml]:
```suggestion
<tool id="transit_hmm" name="HMM" version="@VERSION@+galaxy0">
```

File: tools/transit/transit_resampling.xml [hunk 5]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<tool id="transit_resampling" name="Resampling" version="@VERSION@+galaxy2">
+<tool id="transit_resampling" name="Resampling" version="@VERSION@+galaxy1">
```
Comment (mvdbeek) [inline tools/transit/transit_resampling.xml]:
```suggestion
<tool id="transit_resampling" name="Resampling" version="@VERSION@+galaxy0">
```

File: tools/transit/transit_tn5gaps.xml [hunk 6]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<tool id="transit_tn5gaps" name="Tn5Gaps" version="@VERSION@+galaxy2">
+<tool id="transit_tn5gaps" name="Tn5Gaps" version="@VERSION@+galaxy1">
```
Comment (mvdbeek) [inline tools/transit/transit_tn5gaps.xml]:
```suggestion
<tool id="transit_tn5gaps" name="Tn5Gaps" version="@VERSION@+galaxy0">
```

**Comment by nsoranzo** [comment]
@Delphine-L For future reference, you can commit all the suggested changes in a single commit by using the "Adding suggestion to batch" button.

**Pull request merged**